README.md

```diff
@@ -26,47 +26,53 @@
 [![Node CI](https://github.com/apache/cordova-cli/workflows/Node%20CI/badge.svg?branch=master)](https://github.com/apache/cordova-cli/actions?query=branch%3Amaster)
 [![codecov.io](https://codecov.io/github/apache/cordova-cli/coverage.svg?branch=master)](https://codecov.io/github/apache/cordova-cli?branch=master)
 
-> The command line tool to build, deploy and manage [Cordova](http://cordova.apache.org)-based applications.
+> The command line tool to build, deploy and manage [Cordova](https://cordova.apache.org)-based applications.
 
-[Apache Cordova](http://cordova.apache.org) allows for building native mobile applications using HTML, CSS and JavaScript. 
+[Apache Cordova](https://cordova.apache.org) allows for building native mobile applications using HTML, CSS and JavaScript.
 This tool helps with management of multi-platform Cordova applications as well as Cordova plugin integration.
 
-# Installation
-In your command-line on Windows:    
-```bash    
+## Installation
+
+In your command-line on Windows:
+
+```bash
     c:\> npm install -g cordova
-```    
-    
+```
+
 In your terminal on Mac OS X/Linux:
-```bash    
+
+```bash
     $sudo npm install -g cordova
 ```
 
-# Creating a new Cordova project
+## Creating a new Cordova project
+
 This simple example demonstrates how Cordova CLI can be used to create a `myApp` project with the `camera` plugin and run it for `android` platform:
 
 ```bash
     cordova create myApp com.myCompany.myApp myApp
     cd myApp
     cordova plugin add cordova-plugin-camera --save
     cordova platform add android --save
-    cordova requirements android    
+    cordova requirements android
     cordova build android --verbose
     cordova run android
 ```
 
-# Docs
+## Docs
+
 - [Overview of Cordova]
 - [Create your first Cordova app] guide
-- [Full reference docs for Cordova CLI][Reference docs] has details of commands to add platforms, add plugins, build, package, and sign your HTML, JS apps. 
+- [Full reference docs for Cordova CLI][Reference docs] has details of commands to add platforms, add plugins, build, package, and sign your HTML, JS apps.
 - Cordova allows you to build apps for a number of platforms. Learn more about our [Supported platforms].
 - [Project directory structure] documents the details of the directory structure created by Cordova CLI.
 
-# Contributing
-Cordova is an open source Apache project and contributors are needed to keep this project moving forward. Learn more on 
-[how to contribute on our website][contribute]. 
+## Contributing
+
+Cordova is an open source Apache project and contributors are needed to keep this project moving forward. Learn more on
+[how to contribute on our website][contribute].
```

Comment:
This links does not work I assume.

Reply:
This link works, it is a reference link.